package.json

```diff
@@ -61,7 +61,7 @@
         "@types/chai": "^4.1.5",
         "@types/fs-extra": "^5.0.4",
         "@types/mocha": "^5.2.5",
-        "@types/node": "11.9.1",
+        "@types\node": "11.9.1",
```

Comment:
Same issue as above. This is escaping the n character. This should be a forward slash.